src/components/Toolbar/Toolbar.js

```diff
@@ -41,7 +41,7 @@ import { resourceTypeSelector } from '../../selectors/filterMarkersSelectors';
 import ResourceMenu from '../ResourceMenu/ResourceMenu';
 import NavigationItem from './NavigationItem';
 import useIsMobile from 'hooks/useIsMobile';
-import selectFilteredResource from '../../selectors/waterSelectors';
+import selectFilteredResource from 'selectors/resourceSelectors';
```

Comment:
What is the error you are getting in the docker container? I don't think this is correct because this will attempt to search for a npm-installed package of this name, rather than from a local file.

Reply:
I believe the import (after my change) was resolving to the local module. However, even if that is correct, relative imports are probably better/safer here, true, so I have added a commit to do that. I also went ahead and updated the previous import in the code, which was also not using "dot notation" relative imports. Please let me know if I've missed something. 

I am currently on the go so I will have to recheck what the exact error was in a bit, but I believe the error pointed out the broken import (for the water selector file, which I believe was removed).

Reply:
Here is a screenshot of the exact error.

![image](https://github.com/user-attachments/assets/0819f4a6-713e-4e7e-ad37-6af4f0e49665)

Reply:
> ...this will attempt to search for a npm-installed package of this name, rather than from a local file.

I think that's right for node's resolution process. On the other hand, we are currently using webpack via react-scripts for the build, which has its own resolution process, and I am guessing that's why the import worked even without a "dot" relative import. Though, I am not sure what the default webpack config used by react-scripts is, so I would have to dig a bit more to confirm this.